diff --git a/node_modules/react-native-auto-height-image/AnimatableImage.js b/node_modules/react-native-auto-height-image/AnimatableImage.js
index d38d539..d4fde7d 100644
--- a/node_modules/react-native-auto-height-image/AnimatableImage.js
+++ b/node_modules/react-native-auto-height-image/AnimatableImage.js
@@ -16,8 +16,4 @@ function AnimatableImage(props) {
 
 AnimatableImage.propTypes = Image.propTypes | Animated.Image.propTypes;
 
-AnimatableImage.defaultProps = {
-  animated: false
-};
-
 export default AnimatableImage;
diff --git a/node_modules/react-native-auto-height-image/AutoHeightImage.js b/node_modules/react-native-auto-height-image/AutoHeightImage.js
index 9a0e45a..07232d7 100644
--- a/node_modules/react-native-auto-height-image/AutoHeightImage.js
+++ b/node_modules/react-native-auto-height-image/AutoHeightImage.js
@@ -13,16 +13,16 @@ import { NOOP, DEFAULT_HEIGHT } from './helpers';
 // remove `resizeMode` props from `Image.propTypes`
 const { resizeMode, ...ImagePropTypes } = AnimatableImage.propTypes;
 
-function AutoHeightImage(props) {
-  const {
-    onHeightChange,
-    source,
-    width,
-    style,
-    maxHeight,
-    onError,
-    ...rest
-  } = props;
+function AutoHeightImage({
+  source,
+  width,
+  style,
+  onError,
+  maxHeight = Infinity,
+  onHeightChange = NOOP,
+  animated = false,
+  ...rest
+}) {
   const [height, setHeight] = useState(
     getImageSizeFitWidthFromCache(source, width, maxHeight).height ||
       DEFAULT_HEIGHT
@@ -82,10 +82,4 @@ AutoHeightImage.propTypes = {
   animated: PropTypes.bool
 };
 
-AutoHeightImage.defaultProps = {
-  maxHeight: Infinity,
-  onHeightChange: NOOP,
-  animated: false
-};
-
 export default AutoHeightImage;
diff --git a/node_modules/react-native-auto-height-image/ImagePolyfill.js b/node_modules/react-native-auto-height-image/ImagePolyfill.js
index f0018f4..0e9a4f9 100644
--- a/node_modules/react-native-auto-height-image/ImagePolyfill.js
+++ b/node_modules/react-native-auto-height-image/ImagePolyfill.js
@@ -28,6 +28,5 @@ function ImagePolyfill(props) {
 }
 
 ImagePolyfill.propTypes = AnimatableImage.propTypes;
-ImagePolyfill.defaultProps = AnimatableImage.defaultProps;
 
 export default ImagePolyfill;
